CONTRIBUTORS.md

```diff
@@ -55,6 +55,7 @@ create a pull request.
 | ZE TIAN, BO-JIU | [Leon777-coder](https://github.com/Leon777-coder) | National Yang Ming Chiao Tung University (NYCU) | N |
 | Laura Spang | [LaumiH](https://github.com/LaumiH) | TU Darmstadt | N |
 | Kunihiro Ishiguro | [kishiguro](https://github.com/kishiguro) | Hash-Set | N |
+| Leonardo Azalim de Oliveira | [oliveiraleo](https://github.com/oliveiraleo) |  | N |
```

Comment:
Hi @oliveiraleo , 

I suggest adding your profile to our CONTRIBUTORS! 
Thanks for your contribution. 

I'm unsure about the name and details, so please inform me immediately if there are any issues! Thank you!

Reply:
Hello @andy89923, 

My name and username are correct. In the company/university, please, put as `Federal University of Juiz de Fora (UFJF)` (as displayed on its [website](https://www2.ufjf.br/international/) and on [Wikipedia](https://en.wikipedia.org/wiki/Federal_University_of_Juiz_de_Fora))

Best regards,
Leo.

Reply:
Sure, thanks for the correction.